Server/Components/PlayerMovement.h

```diff
@@ -27,6 +29,18 @@ class PlayerMovement : public Component {
     void Jump();
     void Grapple();
 
+    struct CameraAnimationCalls {
+        bool jump = false;
+        bool isGrounded = false;
+        float land = 0.0f;
+        float fallDistance = 0.0f;
+        float groundMovement = 0.0f;
+        float strafeSpeed = 0.0f;
+        float shakeIntensity = 0.0f;
+    };
+
+    CameraAnimationCalls cameraAnimationCalls;
```

Comment:
Maybe call this CameraAnimationCallData, since this is just a struct and doesnt actually handle the calls.